src/basic/socket-util.c

```diff
@@ -661,27 +661,31 @@ int sockaddr_pretty(
                         /* The name must have at least one character (and the leading NUL does not count) */
                         p = strdup("<unnamed>");
                 else {
+                        /* Note that we calculate the path pointer here through the .un_buffer field in order to trick
+                         * out bounds checking tools such as ubsan, which are too smart for their own good: on Linux
```

Comment:
trick out → outtrick

https://www.merriam-webster.com/dictionary/trick%20out. The word is pretty old, I think 17th or 18th century.

also "than the declared size of the field"